src/__tests__/page-header/button-toolbar/ButtonToolBar.test.tsx

```diff
@@ -17,17 +17,17 @@ describe('Button Tool Bar', () => {
     ]
     jest.spyOn(ButtonBarProvider, 'useButtons').mockReturnValue(buttons)
 
-    const wrapper = mount(<ButtonToolBar />).find('.button-toolbar')
+    render(<ButtonToolBar />)
+    const renderedButtons = screen.getAllByRole('button')
 
-    expect(wrapper.childAt(0).getElement()).toEqual(buttons[0])
-    expect(wrapper.childAt(1).getElement()).toEqual(buttons[1])
+    expect(renderedButtons[0]).toHaveTextContent(/test 1/i)
+    expect(renderedButtons[1]).toHaveTextContent(/test 2/i)
```

Comment:
In this case, I think it would be preferable to have exact value match. It is important that this component does not modify what it was given since this could be a patient name, the name of some procedure, etc.

Reply:
Fixed 😄